core/src/main/java/com/linecorp/armeria/common/HttpResponseWriter.java

```diff
@@ -192,10 +192,7 @@ default void respond(HttpStatus status, MediaType mediaType, HttpData content,
 
     /**
      * Writes the specified HTTP response and closes the stream.
-     *
-     * @deprecated Use {@link HttpResponse#of(AggregatedHttpMessage)}.
      */
-    @Deprecated
     default void respond(AggregatedHttpMessage res) {
```

Comment:
I'm curious if we should rename this method to `close` as well while having a deprecated `respond(AggregatedHttpMessage)`. What do you think?

Reply:
Makes sense, it's more consistent that way